src/pages/workspaces/workspace/Notebooks.js

```diff
@@ -42,6 +46,14 @@ const notebookCardCommonStyles = listView => _.merge({ display: 'flex' },
     }
 )
 
+const galaxyCardStyle = () => {
```

Comment:
nit: Since this is only used in one place and is pretty small, we could inline it